tests/units/iop/test_iop_parser.py

```diff
@@ -293,3 +300,27 @@ def test_wrong_page_nr_value_log(shared_datadir, parser):
                 "log_level": "error",
             },
         ]
+
+
+def test_abstract(shared_datadir, parser):
+    content = (shared_datadir / "abstract.xml").read_text()
+    article = parse_to_ET_element(content)
+    parsed_article = parser._publisher_specific_parsing(article)
+    assert (
+        parsed_article["abstract"]
+        == r'Solar, terrestrial, and supernova neutrino experiments are subject to muon-induced radioactive background. The China Jinping Underground Laboratory (CJPL), with its unique advantage of a 2400 m rock coverage and long distance from nuclear power plants, is ideal for MeV-scale neutrino experiments. Using a 1-ton prototype detector of the Jinping Neutrino Experiment (JNE), we detected 343 high-energy cosmic-ray muons and (7.86<inline-formula xmlns:ns0="http://www.w3.org/1999/xlink"><tex-math>$ \pm $</tex-math><inline-graphic ns0:href="cpc_46_8_085001_M1.jpg" ns0:type="simple" /></inline-formula>3.97) muon-induced neutrons from an 820.28-day dataset at the first phase of CJPL (CJPL-I). Based on the muon-induced neutrons, we measured the corresponding muon-induced neutron yield in a liquid scintillator to be<inline-formula xmlns:ns0="http://www.w3.org/1999/xlink"><tex-math>$(3.44 \pm 1.86_{\rm stat.}\pm $</tex-math><inline-graphic ns0:href="cpc_46_8_085001_M2.jpg" ns0:type="simple" /></inline-formula><inline-formula xmlns:ns0="http://www.w3.org/1999/xlink"><tex-math>$ 0.76_{\rm syst.})\times 10^{-4}$</tex-math><inline-graphic ns0:href="cpc_46_8_085001_M2-1.jpg" ns0:type="simple" /></inline-formula>&#956;<sup>&#8722;1</sup>g<sup>&#8722;1</sup>cm<sup>2</sup>at an average muon energy of 340 GeV. We provided the first study for such neutron background at CJPL. A global fit including this measurement shows a power-law coefficient of (0.75<inline-formula xmlns:ns0="http://www.w3.org/1999/xlink"><tex-math>$ \pm $</tex-math><inline-graphic ns0:href="cpc_46_8_085001_M3.jpg" ns0:type="simple" /></inline-formula>0.02) for the dependence of the neutron yield at the liquid scintillator on muon energy.'
```

Comment:
black? and why `r`? :)

Reply:
because otherwise I am getting the `DeprecationWarning: invalid escape sequence \p`

Reply:
could you please splitted in multiple lines?